tests/braille-specs/de-g2-blista-dictionary.yaml

```diff
@@ -191,3 +191,33 @@ tests:
 - [Quichotte, ⠠⠟⠥⠼⠕⠞⠦] # according to SBS
 - [Quichotte, ⠠⠟⠥⠊⠹⠕⠞⠦, {xfail: true}] # according to blista
 - [ultrarational, ⠥⠇⠞⠗⠁⠗⠐⠝⠁⠇, {xfail: true}] # 136-123-2345-1235-1-1235-5-1345-1-123
+
+
+# tests related to contraction issues (german), October 2023
+
+display: unicode.dis
+table:
+  locale: de
+  grade: 1
+  __assert-match: de-g1.ctb
+tests:
+- [Blista, ⠠⠃⠇⠊⠎⠞⠁, {xfail: true}] # Since it is an abbreviation of lower case letters, it must be announced as lower case and nothing may be contracted there
```

Comment:
This is a grade 1 test but the file is called de-**g2**-blista-dictionary.yaml.

Reply:
I renamed the file